### PR TITLE
Update msf/local_map

### PR DESCRIPTION
**Pull request by tthhee**

### CODE REVIEW

File: modules/localization/msf/common/util/file_utility.cc [hunk 1]
```diff
@@ -0,0 +1,335 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
+ *
```
Comment (jinghaomiao) [inline modules/localization/msf/common/util/file_utility.cc]: 2019

File: modules/localization/msf/common/util/file_utility.h [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+  /**
+   * @brief Check if the directory specified by directory_path exists
+   *        and is indeed a directory.
+   * @param directory_path Directory path.
+   * @return If the directory specified by directory_path exists
+   *         and is indeed a directory.
+   */
+  static bool DirectoryExists(const std::string& directory_path);
+
```
Comment (jinghaomiao) [inline modules/localization/msf/common/util/file_utility.h]: Please use the same function in Cyber to reduce duplicate implementations

**Comment by jinghaomiao** [comment]
Please also rename the commit description to follow Module: description format

File: docs/howto/how_to_run_NDT_localization_module_on_your_local_computer.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+channel: /apollo/sensor/velodyne64/compensator/PointCloud2
+```
+## 3. Run the multi-sensor fusion localization module
+run the script in apollo directory
+```
+./scripts/ndt_localization.sh
+```
+This script will run localization program in the background. You can check if the program is running by using the command.
+```
+ps -e | grep "ndt_localization".
```
Comment (mickeyouyou) [inline docs/howto/how_to_run_NDT_localization_module_on_your_local_computer.md]: maybe should remove the dot of endline.

**Pull request closed without merging**